apps/prairielearn/src/lib/db-types.ts

```diff
@@ -88,6 +88,8 @@ export const ModeSchema = z.enum(['Public', 'Exam', 'SEB']);
 export type Mode = z.infer<typeof ModeSchema>;
 
 export const CourseSchema = z.object({
+  banner_color: z.string().nullable(),
+  banner_html: z.string().nullable(),
```

Comment:
We could name these `announcement_*` for consistency with the global one?

Reply:
I'm (weakly) in favor of that.

Reply:
Changed in da38600.